packages/office-ui-fabric-react/src/components/ProgressIndicator/ProgressIndicator.styles.ts

```diff
@@ -29,7 +29,7 @@ const IndeterminateProgressRTL = keyframes({
 });
 
 export const getStyles = (props: IProgressIndicatorStyleProps): IProgressIndicatorStyles => {
-  const isRTL = getRTL();
+  const isRTL = getRTLWithTheme(props.theme);
```

Comment:
You're already getting `theme` from props below, you could move this line and have this instead:

```suggestion
  const isRTL = getRTLWithTheme(theme);
```